src/memote/support/consistency.py

```diff
@@ -20,15 +20,14 @@
 from __future__ import absolute_import, division
 
 import logging
+import multiprocessing
 from operator import attrgetter
 
 import numpy as np
-import multiprocessing
-from cobra import Reaction
+from cobra import Configuration, Reaction
 from cobra.exceptions import Infeasible
 from cobra.flux_analysis import flux_variability_analysis
-from cobra import Configuration
-from optlang.interface import OPTIMAL, INFEASIBLE
+from optlang.interface import INFEASIBLE, OPTIMAL
```

Comment:
A change like this is super arcane to me. Does it matter in which order these two are imported?

Reply:
Honestly, I simply ran `isort` on the module that did this.